docs/0.2-DRAFT/index.md

```diff
@@ -87,7 +81,7 @@ _RO-Crate JSON-LD_: JSON-LD using the _RO-Crate JSON-LD Context_ containing RO-C
 
 Throughout this specification, RDF terms are referred to using the keys defined in RO-Crate JSON-LD Context.
 
-Following [schema.org](http://schema.org) practice, property names start with lowercase letters and class names with Uppercase.
+Following [schema.org](http://schema.org) practice, property names start with lowercase letters and class names start with uppercase letters.
```

Comment:
this was intended as an inline demonstration.. perhaps we can use _`property` names_ and _`Class` names_